department-of-veterans-affairs/abd-vro
# Database changes and Java logic

## What was the problem?


Associated tickets or Slack threads:

- [MCP-2303](https://amida.atlassian.net/browse/MCP-2303)

## How does this fix it?[^1]
Populates the veteran flash ID table

## How to test this PR
- Run the automated claim endpoint using a valid integer represented as a string for veteranFlashIds.
- Check the database and verify that flash ids are populated in the flash id table and that the veteranIcn matches the request.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
